Samples/Swift/DaysUntilBirthday/Sources/ContentView.swift

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import GoogleSignIn
+
+struct ContentView: View {
+  @EnvironmentObject var authViewModel: AuthenticationViewModel
+
+  var body: some View {
+    return Group {
+      NavigationView {
+        switch authViewModel.state {
+        case .signedIn:
+          UserProfileView()
+            .navigationTitle(
+              NSLocalizedString(
+                "User Profile",
+                comment: "User profile navigation title"
+              ))
+        case .signedOut:
+          SignInView()
+            .navigationTitle(
+              NSLocalizedString(
+                "Sign-in with Google",
+                comment: "Sign-in navigation title"
+              ))
+        }
+      }
+      .navigationViewStyle(.stack)
```

Comment:
Reminder about the apparent Xcode 12 build issue at this line.  Should we make this Sample Xcode 13+ only and document it, or go for backward compatibility?

Reply:
I think it's best to avoid the issue and use `StackNavigationViewStyle`.